compiler/include/symbol.h

```diff
@@ -893,4 +893,9 @@ const char* toString(Symbol* sym, bool withTypeAndIntent);
 typedef std::vector<std::pair<Symbol*, Symbol*> > SymbolMapVector;
 SymbolMapVector sortedSymbolMapElts(const SymbolMap& map);
 
+class SymbolMapKeyValue { public: Symbol *key, *value;
+  SymbolMapKeyValue(Symbol* k, Symbol* v): key(k), value(v) { } };
+typedef std::vector<SymbolMapKeyValue> SymbolMapKVvector;
+SymbolMapKVvector sortedSymbolMapKVs(const SymbolMap& map);
```

Comment:
I'm not following why this new code is here since it seems the same as sortedSymbolMapElts. Are you asking if I like this structure better than the pair?

Reply:
Correct. This way I do not need to declare `Symbol* key = pair.first`, I can just say `elem.key` wherever I need the key.

Reply:
It's fine with me to switch from the pair to the struct but let's keep the old names `SymbolMapVector` and `sortedSymbolMapElts`. Also let's declare it as a struct rather than a class with public fields.